service/src/main/java/uk/gov/hmcts/reform/fpl/validation/validators/IsStateMigratableValidator.java

```diff
@@ -23,6 +23,6 @@ public boolean isValid(CaseData caseData, ConstraintValidatorContext constraintV
     }
 
     private boolean hasFinalFields(Child child) {
-        return child.getFinalOrderIssuedType() != null || child.getFinalOrderIssued() != null;
+        return child.getFinalOrderIssuedType() != null || "Yes".equals(child.getFinalOrderIssued());
```

Comment:
Has it been checked that `child.getFinalOrderIssuedType() != null` is a valid condition still?

Reply:
yup discussed / tested and believe this condition is still valid / works fine